cloudify_hostpool/tests/storage/test_sqlite.py

```diff
@@ -0,0 +1,161 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import os
+import tempfile
+import unittest
+
+from cloudify_hostpool.storage import sqlite
+
+
+class SQLiteTest(unittest.TestCase):
+    db = None
+    tempfile = None
+    server_list = None
+
+    def setUp(self):
+        _, self.tempfile = tempfile.mkstemp()
+        self.db = sqlite.SQLiteStorage(self.tempfile)
+
+    def tearDown(self):
+        if self.tempfile is not None:
+            os.unlink(self.tempfile)
+            self.tempfile = None
```

Comment:
why create and destroy the data base for each test? maybe once per class is enough? 
i mean use the setupClass and teardownClass